docs/semgrep-ci/overview.md

```diff
@@ -1,262 +1,155 @@
-When connected to Semgrep App, you can also add non-blocking rules to your scans.
-Non-blocking rules return [non-blocking findings](#getting-notifications-instead-of-blocking-builds) which notify you via an integration but do not show up in log output, and do not cause jobs to fail with a ❌.
-:::
+This section describes a high-level view of steps to integrate Semgrep with Semgrep App.
 
-### Integrations
+![High level view of steps to integrate and refine Semgrep in your CI environment with Semgrep App](/img/semgrep-ci-overview-app.png "High level view of steps to integrate and refine Semgrep in your CI environment with Semgrep App")
 
-For users of Semgrep App, you can integrate Semgrep CI with many other services, to get you results in the workflow you're already used to, whether you're a developer or part of a security team.
+_Figure 1._ High level view of steps to integrate and refine Semgrep in your CI environment with Semgrep App.
 
-#### Notifications
+Integrating Semgrep in your CI environment through Semgrep App is the **recommended method** for CI providers such as GitHub Actions and GitLab CI/CD. 
 
-![Screenshot of a Slack notification describing the details of a finding](/img/slack-notification.png)<br />
-A Slack notification triggered by new findings in a pull request
+The following steps outline the general procedure to automate Semgrep in many CI environments wish Semgrep App.
 
-Notifications require connection to Semgrep App. You can get notified about new findings via:
+1. [Sign in](https://semgrep.dev/login) to Semgrep App.
+2. Click on **Projects > Scan New Project**.
+3. Follow instructions in the App. These steps vary based on your CI provider.
+4. Semgrep App generates a `SEMGREP_APP_TOKEN` and creates a configuration file. 
+5. Commit the configuration file and add `SEMGREP_APP_TOKEN` as a credential, token, or secret.
+6. The scan automatically begins.
+7. After the scan, Semgrep App reports findings (if any) into the Findings dashboard.
+8. Optional: If you are not satisfied with your CI job's behavior, troubleshoot and edit the configuration file. Refer to [Sample CI configuration for Semgrep App](semgrep-ci/sample-ci-configs).
+9. Repeat from step 2 to integrate Semgrep into remaining repositories.
 
-- [GitHub pull request comments](/semgrep-app/notifications/#github-pull-request-comments)
-- GitLab merge request comments ([sign up for the beta here](https://go.r2c.dev/join-gitlab-beta))
-- [Slack messages](/semgrep-app/notifications/#slack)
-- [emails](/semgrep-app/notifications/#email)
-- [webhooks](/semgrep-app/notifications/#webhooks) (paid feature in Semgrep App)
+#### Default behaviors of scans set up through Semgrep App
 
-To set up notifications, see [Notifications document](/semgrep-app/notifications) for more information.
+* Findings do not block a PR or MR. The presence of findings can be customized to block a PR or MR through the [Rule Board](docs/semgrep-app/rule-board).
```

Comment:
I got a 404 when following this link. I think the one you want is `/docs/semgrep-app/rule-board/`.